tests/failpoints.rs

```diff
@@ -15,6 +15,7 @@
 #![feature(fnbox)]
 #![feature(slice_patterns)]
 #![feature(box_syntax)]
+#![feature(test)]
```

Comment:
do we need this feature here?

Reply:
this PR includes `mod storage`, and there are some bench test inside `mod storage`.